### PR TITLE
Update CAS warning alert box

### LINKED ISSUE #11625: Need to document language version
Can we update this document to include that partial trust is only supported until C# 7.0. Any version beyond 7.0 is focused on .NET Core features and is not guaranteed to emit IL that is compatible with partial trust.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b136e1b9-ac93-6fc4-616a-c24c6acf4945
* Version Independent ID: b8e47e7a-4e2e-fc0a-b0e3-20d219e97dac
* Content: [Code Access Security](https://docs.microsoft.com/en-us/dotnet/framework/misc/code-access-security#feedback)
* Content Source: [docs/framework/misc/code-access-security.md](https://github.com/dotnet/docs/blob/master/docs/framework/misc/code-access-security.md)
* Product: **dotnet-framework**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #11625

### CODE REVIEW

File: docs/framework/misc/code-access-security-basics.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ ms.assetid: 4eaa6535-d9fe-41a1-91d8-b437cfc16921
 
 # Code Access Security Basics
 
-[!INCLUDE[net_security_note](../../../includes/net-security-note-md.md)]
+[!INCLUDE[net_security_note](../../../includes/net-security-note-md.md)]
```
Comment (Youssef1313) [inline docs/framework/misc/code-access-security-basics.md:15]: Was the two spaces here intentional?

```suggestion
[!INCLUDE[net_security_note](../../../includes/net-security-note-md.md)]
```

File: docs/framework/misc/dangerous-permissions-and-policy-administration.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ helpviewer_keywords:
 ms.assetid: 1929e854-23a0-4bb1-94be-e8aa3b609e32
 ---
 # Dangerous Permissions and Policy Administration
+
+[!INCLUDE[net_security_note](../../../includes/net-security-note-md.md)]
```
Comment (Youssef1313) [inline docs/framework/misc/dangerous-permissions-and-policy-administration.md:15]:
```suggestion
[!INCLUDE[net_security_note](../../../includes/net-security-note-md.md)]
```

File: docs/framework/misc/securing-exception-handling.md [hunk 3]
```diff
@@ -12,6 +12,9 @@ helpviewer_keywords:
 ms.assetid: 1f3da743-9742-47ff-96e6-d0dd1e9e1c19
 ---
 # Securing Exception Handling
+
+[!INCLUDE[net_security_note](../../../includes/net-security-note-md.md)]
```
Comment (Youssef1313) [inline docs/framework/misc/securing-exception-handling.md:16]:
```suggestion
[!INCLUDE[net_security_note](../../../includes/net-security-note-md.md)]
```

File: docs/framework/misc/security-and-public-read-only-array-fields.md [hunk 4]
```diff
@@ -7,6 +7,9 @@ helpviewer_keywords:
 ms.assetid: 3df28dee-2a9f-40ff-9852-bfdbe59c27f3
 ---
 # Security and Public Read-only Array Fields
+
+[!INCLUDE[net_security_note](../../../includes/net-security-note-md.md)]
```
Comment (Youssef1313) [inline docs/framework/misc/security-and-public-read-only-array-fields.md:11]:
```suggestion
[!INCLUDE[net_security_note](../../../includes/net-security-note-md.md)]
```

File: docs/framework/misc/security-and-remoting-considerations.md [hunk 5]
```diff
@@ -10,6 +10,9 @@ helpviewer_keywords:
 ms.assetid: 125d2ab8-55a4-4e5f-af36-a7d401a37ab0
 ---
 # Security and Remoting Considerations
+
+[!INCLUDE[net_security_note](../../../includes/net-security-note-md.md)]
```
Comment (Youssef1313) [inline docs/framework/misc/security-and-remoting-considerations.md:14]:
```suggestion
[!INCLUDE[net_security_note](../../../includes/net-security-note-md.md)]
```

File: docs/framework/misc/security-and-serialization.md [hunk 6]
```diff
@@ -14,6 +14,8 @@ ms.assetid: b921bc94-bd3a-4c91-9ede-2c8d4f78ea9a
 ---
 # Security and serialization
 
+[!INCLUDE[net_security_note](../../../includes/net-security-note-md.md)]
```
Comment (Youssef1313) [inline docs/framework/misc/security-and-serialization.md:17]:
```suggestion
[!INCLUDE[net_security_note](../../../includes/net-security-note-md.md)]
```

**Comment by adegeo** [comment]
I believe the first paragraph under the H1 is indexed for SEO, so we'll lose the original intro paragraph SEO impact.

**Review by BillWagner — APPROVED** [review]
This LGTM @tdykstra 

You can :shipit: when ready.

**Comment by tdykstra** [comment]
@adegeo Good point, but in this case I think the need to stress the content of the alert outweighs any SEO benefit we might gain by locating it farther down the page.

**Pull request merged**